runtime/integration-tests/build.rs

```diff
@@ -0,0 +1,114 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
+//
+// This file is part of the Centrifuge chain project.
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+use std::{env, fs, path::PathBuf, process::Command};
+
+const LP_SOL_SOURCES: &str = "LP_SOL_SOURCES";
+
+fn main() {
+	let paths = fs::read_dir("./submodules/")
+		.expect("Submodules directory must exist for integration-tests");
+	let out_dir = env::var("OUT_DIR").expect("Cargo sets OUT_DIR environment variable. qed.");
+
+	/*
+	match Command::new("git")
+		.args(&["fetch", "--all", "--recurse-submodules=yes"])
+		.output()
+	{
+		Ok(o) if o.status.success() => {}
+		Ok(o) => {
+			println!(
+				"cargo:warning=Git fetch failed with: \n  - status: {}\n   -stderr: {}",
+				o.status,
+				String::from_utf8(o.stderr).expect("stderr is utf-8 encoded. qed.")
+			);
+		}
+		Err(err) => {
+			println!("cargo:warning=Failed to execute git command: {}", err);
+		}
+	}
+	 */
```

Comment:
+1. AFAICT, we should solve this via CI which regularly pulls the latest contracts

Reply:
> we should solve this via CI which regularly pulls the latest contracts

One concern regarding this is that an update in `liquidity-pools` can break our CI. I suppose it is `centrifuge-chain` who should choose the commit used from `liquidity-pools`. When liquidity-pools is updated, then we should create a commit in `centrifuge-chain` just updating the dependency.

Reply:
I am not 100% sure on the process here, tbh. @hieronx wdyt?

Reply:
> When liquidity-pools is updated, then we should create a commit in centrifuge-chain just updating the dependency.

I think this makes sense. Also because we probably want to be able to run the main tests against the release commit from liquidity pools (what is live now), but also want to be able to run the tests in a branch against a more recent development commit. Having the commit be chosen manually would make this possible.

Reply:
IMO, the Liquidity Pools repo should include a CI which runs LP integration tests from Centrifuge Chain and notify Protocol / Centrifuge Chain repo when it is failing such that we know a breaking change has been introduced on the EVM side.